src/features/registry.js

```diff
@@ -41,23 +43,40 @@ systemJSPrototype.set = function (id, module) {
 
 systemJSPrototype.has = function (id) {
   const load = this[REGISTRY][id];
-  return load && load.e === null && !load.E;
+  return !!load;
```

Comment:
👍  for forcing a boolean return value.

The other changes seem to be possibly breaking? Will this result in situations where the following code would no longer work?

```js
if (System.has('http://localhost:8080/hi.js')) {
  // With the new changes, there's a chance that this returns undefined now
  // because the module hasn't finished instantiating?
  const m = System.get('http://localhost:8080/hi.js')
}
```

Reply:
Get already returns `null` for modules in an error state, so the code path would already need a falsy check. We just have both null and undefined now for an error and not existing respectively.

I doubt anyone would be relying on this behaviour.

Reply:
sounds good